backend/test/integration/run_tests_locally.sh

```diff
@@ -32,7 +32,22 @@ case "$response" in
         ;;
 esac
 
-echo "Starting integration tests..."
-command="go test -v ./... -namespace ${NAMESPACE} -args -runIntegrationTests=true -isDevMode=true"
-echo $command "$@"
-$command "$@"
+if [ "$1" == "postgres" ]; then
+    echo "Starting PostgreSQL DB port forwarding..."
+    kubectl -n "$NAMESPACE" port-forward svc/postgres-service 5432:5432 --address="127.0.0.3" & PORT_FORWARD_PID=$!
+    # wait for kubectl port forward
+    sleep 10
+    echo "Starting integration tests..."
+    command="go test -v ./... -namespace ${NAMESPACE} -args -runIntegrationTests=true -isDevMode=true -runPostgreSQLTests=true"
```

Comment:
Looks like only this line is different between `if` and `else` condition, can we append the `-runPostgreSQLTests=true` to the command if this is postgresql, so we can reduce some duplication.

Reply:
Thank you for finding this. The command for port forwarding is also slightly different for these two DBs. About 4 lines above. I moved the more repetitive lines outside so the script is more succinct.